mlir/include/mlir/Dialect/Linalg/TransformOps/LinalgTransformOps.td

```diff
@@ -2347,6 +2347,9 @@ def VectorizeChildrenAndApplyPatternsOp :
     operation that is contained inside the vectorization target.
 
     This transformation supports the following attributes:
+    - `vectorize_mixed_precision`: a `UnitAttr` to activate the vectorization
```

Comment:
```suggestion
    - `fold_mixed_precision_into_contract`: a `UnitAttr` to activate the vectorization
```

IIUC, vectorization will happen regardless of this this attribute. Could you update the comment accordingly? Thanks!